src/sentry/models/avatars/user_avatar.py

```diff
@@ -46,5 +50,39 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_useravatar"
 
+    def outboxes_for_update(self, shard_identifier: int | None = None) -> List[ControlOutboxBase]:
```

Comment:
Could `UserAvatar` mixin be used on this model?

Reply:
I tried many variations to get this to work and failed -- ultimately, all of these subclasses all want to be django model subclasses so they have a sensible `super` path, but django orm itself complains due to the `id` field conflict and other things.